### PR TITLE
feat: ability to customize file name

### PR DESCRIPTION
**Pull request by kirillzyusko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,9 +80,9 @@ cd ios && pod install
 
 Synchronously starts the Hermes Profiling Session.
 
-### `stopProfiling(saveInDownloadsDirectory: boolean): Promise<string>`
+### `stopProfiling(saveInDownloadsDirectory: boolean, fileName: string): Promise<string>`
```
Comment (hannojg) [inline README.md:83]: this is an optional, right? Let's add a `?` to the param type here

Comment (kirillzyusko) [reply README.md:83]: @hannojg actually `saveInDownloadsDirectory` is also optional (false by default), but it's not marked as optional in README. Should we mark both params as optional then?

Comment (hannojg) [reply README.md:83]: I think we should yes!

Comment (kirillzyusko) [reply README.md]: Done!

**Review by hannojg — APPROVED** [review]

**Pull request merged**